relay/relay.go

```diff
@@ -498,14 +498,21 @@ func (r *Relay) reSetupMeta() error {
 		return err
 	}
 
+	lastPos, _, err := utils.GetMasterStatus(r.db, r.cfg.Flavor)
```

Comment:
do not support fetching binlog from the newest GTID sets?

Is any better name than `lastPos`? like `latestPos` or `newest`?

Reply:
my mistake, fixed in https://github.com/pingcap/dm/pull/304/commits/36603239528f65c8a27f7cd1c9660cc920d2a65b